src/main/java/com/puppycrawl/tools/checkstyle/api/FullIdent.java

```diff
@@ -136,8 +180,7 @@ private static boolean isArrayTypeDeclaration(DetailAST arrayDeclarator) {
      * @param ast the type ast we are building a {@code FullIdent} for
      */
     private static void appendBrackets(FullIdent full, DetailAST ast) {
-        final int bracketCount =
-                ast.getParent().getChildCount(TokenTypes.ARRAY_DECLARATOR);
+        final int bracketCount = ast.getParent().getChildCount(TokenTypes.ARRAY_DECLARATOR);
```

Comment:
revert or move this not functional formatting to separate commit "minor: formatting"

Reply:
done